backend/de.metas.acct.base/src/main/java/de/metas/acct/gljournal_sap/acct/Doc_SAPGLJournal.java

```diff
@@ -76,27 +76,19 @@ protected List<Fact> createFacts(final AcctSchema as)
 		for (final SAPGLJournalLine line : glJournal.getLines())
 		{
 			final BigDecimal amtSourceDr;
```

Comment:
what's the reason behind those changes wtih Doc_SAPGLJournal?

why `line.getAmountAcct()` isn't used anymore?

Reply:
mainly because of this bug here: https://github.com/metasfresh/me03/issues/15095#issuecomment-1497576610. We let the framework to take care of computing the acct amounts based on source amounts and the other configurations